Don't delete existing file if download fails (#2618)
zowe/zowe-cli: packages/zosfiles/CHANGELOG.md

## Recent Changes

- Enhancement: Added a download `destination` field to the API response for the `Download.ussFile` and `Download.dataSet` functions. [#2641](https://github.com/zowe/zowe-cli/pull/2641)

> - BugFix: Fixed an issue where if a file fails to download and one already exists in the destination, the old file is deleted. [#2618](https://github.com/zowe/zowe-cli/pull/2618)

Reviewer: ```suggestion
- BugFix: Fixed an issue where if a file failed to download and an older version of it already existed in the destination, the old file was deleted. [#2618](https://github.com/zowe/zowe-cli/pull/2618)
```

> - BugFix: Fixed an issue where the preserve letter case option was not preserving the case in generated directory names in the `Download` `data-sets-matching` command. [#2618](https://github.com/zowe/zowe-cli/pull/2618)

Reviewer: ```suggestion
- BugFix: Fixed an issue where the preserve letter case option was not preserving the case in generated directory names in the `download data-sets-matching` command. [#2618](https://github.com/zowe/zowe-cli/pull/2618)
```
